protokt-codegen/src/main/kotlin/protokt/v1/codegen/generate/FileGenerator.kt

```diff
@@ -47,6 +49,12 @@ private class FileGenerator(
                     }.build()
                 )
 
+                addAnnotation(
+                    AnnotationSpec.builder(ClassName.bestGuess("kotlin.OptIn"))
+                        .addMember("$PROTOKT_V1.OnlyForUseByGeneratedProtoCode::class")
```

Comment:
The compiler doesn't actually let you reference these by class literal. Weird language design decision.